### PR TITLE
Enable currency settings and installed sticker synchronisation

### LINKED ISSUE #12889: Sychronize sticker packs across devices
As a user
I want to have my sticker packs synchronized across devices
So I don't have to install them on each device.

### Description

When a user installs a sticker pack, it should also appear on any paired device that they own.
Similarly, if a sticker pack is removed, the same should happen on the paired device.

This settings should also be backed up, similarly as we do for contacts.

### LINKED ISSUE #13190: Synchronise `currency` setting
Implement functionality that handles the syncing of the `currency` setting.

### Background

In `status-go` syncing of settings has been implemented, and additional setting sync *should be* a relatively straightforward process. The syncing of the `currency` setting has been implemented from the `status-go` side.

See https://github.com/status-im/status-go/pull/2478 and https://github.com/status-im/status-react/pull/13053

However there is additional functionality that needs to be implemented on the `status-react` side to make currency work correctly.

See the QA report here [ISSUE 6: Error on the synced device when changing currency](https://github.com/status-im/status-react/pull/13053#issuecomment-1065179963)

### `status-go` Implementation

Currently the `currency` setting has its sync functionality disabled:

```go
Currency = SettingField{
	reactFieldName: "currency",
	dBColumnName:   "currency",
	// TODO resolve issue 6 https://github.com/status-im/status-react/pull/13053#issuecomment-1065179963
	syncProtobufFactory: &SyncProtobufFactory{
		inactive:          true, // Remove after issue is resolved
		fromInterface:     currencyProtobufFactory,
		fromStruct:        currencyProtobufFactoryStruct,
		valueFromProtobuf: StringFromSyncProtobuf,
		protobufType:      protobuf.SyncSetting_CURRENCY,
	},
}
```

To reactivate the `currency` functionality in `status-go` simply remove the `inactive` field from the `SyncProtobufFactory`

### PR DESCRIPTION
**Pull request by audriu**
Closes https://github.com/status-im/status-react/issues/12889
Closes https://github.com/status-im/status-react/issues/13190
Uncomment currency synchronization and sticker pack synchronization.

it is part of the work for this PR:  https://github.com/status-im/status-react/issues/13190

### CODE REVIEW

File: multiaccounts/settings/columns.go [hunk 1]
```diff
@@ -40,7 +40,6 @@ var (
 		dBColumnName:   "currency",
 		// TODO resolve issue 6 https://github.com/status-im/status-react/pull/13053#issuecomment-1065179963
```
Comment (Samyoul) [inline multiaccounts/settings/columns.go]: I'd also remove this line.

**Review by Samyoul — APPROVED** [review]

File: multiaccounts/settings/columns.go (hunk 1 above)
Comment (audriu) [reply multiaccounts/settings/columns.go:41]: oh yes - that makes sense. I will soon.

**Comment by jakubgs** [comment]
I can reproduce the linting issue locally on your branch, but not on `develop`:
```
 > make lint                    
lint
WARN [runner] The linter 'golint' is deprecated (since v1.41.0) due to: The repository of the linter has been archived by the owner.  Replaced by revive. 
protocol/messenger_sync_settings.go:6: File is not `goimports`-ed with -local github.com/ethereum/go-ethereum,github.com/status-im/status-go (goimports)
	"go.uber.org/zap"
make: *** [Makefile:315: lint] Error 1
```

**Comment by jakubgs** [comment]
Looks like a normal linting issue to me, that can be fixed with `go fmt`:
```
~/go/src/github.com/status-im/status-go enable-currency-settings
 > go fmt protocol/messenger_sync_settings.go
protocol/messenger_sync_settings.go

~/go/src/github.com/status-im/status-go enable-currency-settings*
 > g diff
```
```diff
diff --git a/protocol/messenger_sync_settings.go b/protocol/messenger_sync_settings.go
index 224a826c9..472e8be05 100644
--- a/protocol/messenger_sync_settings.go
+++ b/protocol/messenger_sync_settings.go
@@ -3,8 +3,8 @@ package protocol
 import (
        "context"
 
-       "go.uber.org/zap"
        "encoding/json"
+       "go.uber.org/zap"
 
        "github.com/status-im/status-go/multiaccounts/errors"
        "github.com/status-im/status-go/multiaccounts/settings"
```

**Pull request merged**